### PR TITLE
HARP-5900 Add satellite tiles example

### PR DESCRIPTION
**Pull request by musculman**
Add `ppi` to request only if explicitly initialized with a value, otherwise satellite tiles do not work.
Change valid zoom levels of web tiles from 1 to 20.
Signed-off-by: Ignacio Julve <41909512+musculman@users.noreply.github.com>

### CODE REVIEW

File: @here/harp-examples/src/datasource_satellitetile.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * as [[WebTileDataSourceOptions]]
+ * ```typescript
+ * [[include:harp_gl_datasource_satellitetile_1.ts]]
+ * ```
+ * Then added to the [[MapView]]
+ * ```typescript
+ * [[include:harp_gl_datasource_satellitetile_2.ts]]
+ * ```
+ */
+export namespace WebTileDataSourceExample {
```
Comment (ninok) [inline @here/harp-examples/src/datasource_satellitetile.ts]: Please use a different namespace for this example

File: @here/harp-examples/src/datasource_webtile.ts [hunk 2]
```diff
@@ -63,7 +63,8 @@ export namespace WebTileDataSourceExample {
     // snippet:harp_gl_datasource_webtile_1.ts
     const webTileDataSource = new WebTileDataSource({
         appId,
-        appCode
+        appCode,
+        ppi: 320
```
Comment (nzjony) [inline @here/harp-examples/src/datasource_webtile.ts]: this seems to be for a phone, is that the general use case that this example should show?

File: @here/harp-examples/src/datasource_satellitetile.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            canvas,
+            theme: "resources/berlin_tilezen_base.json"
+        });
+        map.setCacheSize(100, 100);
+
+        // instantiate the default map controls, allowing the user to pan around freely.
+        const controls = new MapControls(map);
+
+        // Add an UI.
+        const ui = new MapControlsUI(controls);
```
Comment (nzjony) [inline @here/harp-examples/src/datasource_satellitetile.ts]: should we limit the tilt here as in other examples?

File: @here/harp-examples/src/datasource_satellitetile.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+export namespace WebTileDataSourceExample {
+    // creates a new MapView for the HTMLCanvasElement of the given id
+    export function initializeMapView(id: string): MapView {
+        const canvas = document.getElementById(id) as HTMLCanvasElement;
+
+        const map = new MapView({
+            canvas,
+            theme: "resources/berlin_tilezen_base.json"
+        });
+        map.setCacheSize(100, 100);
```
Comment (nzjony) [inline @here/harp-examples/src/datasource_satellitetile.ts]: why is this needed?

File: @here/harp-examples/src/datasource_webtile.ts (hunk 2 above)
Comment (musculman) [reply @here/harp-examples/src/datasource_webtile.ts]: It is what it was showing before, I think for most of the browsers is the best setting

File: @here/harp-examples/src/datasource_satellitetile.ts (hunk 3 above)
Comment (musculman) [reply @here/harp-examples/src/datasource_satellitetile.ts]: They are webtiles, I don't see why is that needed?

File: @here/harp-examples/src/datasource_satellitetile.ts (hunk 4 above)
Comment (musculman) [reply @here/harp-examples/src/datasource_satellitetile.ts]: Was already in the previous one, so I did not change it
